classes/Kohana/Log.php

```diff
@@ -10,7 +10,7 @@
  * @copyright  (c) 2008-2012 Kohana Team
  * @license    http://kohanaframework.org/license
  */
-class Kohana_Log {
+class Kohana_Log extends Psr\Log\AbstractLogger implements Kohana_Log_Buffer {
 
 	// Log message levels - Windows users see PHP Bug #18090
 	const EMERGENCY = LOG_EMERG;    // 0
```

Comment:
I would prefer going with the last suggestion, to hardcode the integer values. Yes, it removes the inconsistency. And in `Log_Syslog` we're already reverting back to `Log_` values.